app/src/org/commcare/preferences/FormEntryPreferences.java

```diff
@@ -70,20 +50,12 @@ public void onStart() {
 
     @Override
     public void onSharedPreferenceChanged(final SharedPreferences sharedPreferences, final String key) {
-        switch (key) {
-            case KEY_FONT_SIZE:
-                reportEditPreference(GoogleAnalyticsFields.LABEL_FONT_SIZE);
-                updateFontSize();
-                break;
-            case KEY_HELP_MODE_TRAY:
-                reportEditPreference(GoogleAnalyticsFields.LABEL_INLINE_HELP);
+        super.onSharedPreferenceChanged(sharedPreferences, key);
+        if (KEY_FONT_SIZE.equals(key)) {
```

Comment:
Confirming we're intentionally removing these reports

Reply:
Think you noticed this already, but this is now done in the parent class